docs/guides/high-availability/concepts-memory-and-cpu-sizing.adoc

```diff
@@ -61,6 +61,62 @@ For every 100 login/logout/refresh requests per second:
 
 The vCPU requirement is given as a range, as with an increased CPU saturation on the database host the CPU usage per request decreases while the response times increase. A lower CPU quota on the database can lead to slower response times during peak loads. Choose a larger CPU quota if fast response times during peak loads are critical. See below for an example.
 
+=== User event metrics:
+
+This section details the use of the user event metrics such as `login` , `refresh_token` ,
+`client_login`  to monitor and analyze activities related to user logins,
+refresh token requests, and client credential grants respectively.
+
+These metrics are crucial for tracking daily and weekly fluctuations in user activity loads,
+identifying emerging trends that may indicate the need to resize the system and
+validating sizing calculations.
+By systematically measuring and evaluating these user event metrics,
+you can ensure your system remains appropriately scaled and responsive to changes in user behavior and demand.
+
+This is a preview feature and can be enabled with a feature flag.
```

Comment:
I suppose we already put that in the main docs about this one, and I wouldn't name it here again, as would need to update all of these if it is no longer preview.

```suggestion
```